src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -59,36 +59,39 @@ struct AirflowNetworkData;
 
 namespace AirflowNetwork {
 
-    enum VentControlType // TODO: make enum class
+    enum VentControlType // TODO: make enum class, since this enum is used in SetupOutputVariable()
     {
-        None = 0,  // Wrong input
-        Temp = 1,  // Temperature venting control
-        Enth = 2,  // Enthalpy venting control
-        Const = 3, // Constant venting control
-        ASH55 = 4,
-        CEN15251 = 5,
-        NoVent = 6,    // No venting
-        ZoneLevel = 7, // ZoneLevel control for a heat transfer subsurface
-        AdjTemp = 8,   // Temperature venting control based on adjacent zone conditions
-        AdjEnth = 9    // Enthalpy venting control based on adjacent zone conditions
+        None = -1, // Wrong input
+        Temp,      // Temperature venting control
+        Enth,      // Enthalpy venting control
+        Const,     // Constant venting control
+        ASH55,
+        CEN15251,
+        NoVent,    // No venting
+        ZoneLevel, // ZoneLevel control for a heat transfer subsurface
+        AdjTemp,   // Temperature venting control based on adjacent zone conditions
+        AdjEnth    // Enthalpy venting control based on adjacent zone conditions
     };
 
-    enum OpenStatus // TODO: make enum class
+    enum OpenStatus // TODO: make enum class, since this enum is used in SetupOutputVariable()
     {
-        FreeOperation = 0,     // Free operation
-        MinCheckForceOpen = 1, // Force open when opening elapsed time is less than minimum opening time
-        MinCheckForceClose = 2 // Force open when closing elapsed time is less than minimum closing time
+        Uninitialized = -1,
```

Comment:
If it wasn't needed before, I tend to think it is not needed now. I'll check to be certain.